docs/src/main/sphinx/object-storage/legacy-azure.md

```diff
@@ -192,3 +192,59 @@ CREATE TABLE hive.abfs_export.orders_abfs
 WITH (format = 'ORC')
 AS SELECT * FROM tpch.sf1.orders;
 ```
+
+(legacy-azure-migration)=
+## Migration to native Azure implementation
+
+Trino includes a {doc}`native Azure implementation
+</object-storage/file-system-azure>` that does not rely on the legacy Hadoop
+implementation. Trino recommends upgrading existing deployments to the new
```

Comment:
Maybe reword to create more urgency ... strong advised to switch and use the native integration

also mention that legacy will be deprecated and removed